src/lib/icons/IconThumbDown.svelte

```diff
@@ -1,12 +1,28 @@
-<!-- source: https://fonts.google.com/icons?selected=Material%20Icons%3Athumb_down%3A -->
+<!-- source: DFINITY foundation -->
 <svg
+  width="48"
+  height="48"
+  viewBox="0 0 48 48"
+  role="img"
+  aria-label="Dislike or Thumbs Down Icon"
   xmlns="http://www.w3.org/2000/svg"
-  height="18px"
-  viewBox="0 0 24 24"
-  width="18px"
-  fill="currentColor"
-  data-tid="thumb-down"
-  ><path d="M0 0h24v24H0z" fill="none" /><path
-    d="M15 3H6c-.83 0-1.54.5-1.84 1.22l-3.02 7.05c-.09.23-.14.47-.14.73v2c0 1.1.9 2 2 2h6.31l-.95 4.57-.03.32c0 .41.17.79.44 1.06L9.83 23l6.59-6.59c.36-.36.58-.86.58-1.41V5c0-1.1-.9-2-2-2zm4 0v12h4V3h-4z"
-  /></svg
 >
+  <path
+    fill="#F2556F"
```

Comment:
I had the same concern with the theme, but the icon seems to be always the same. Not themable.

<img width="257" alt="Screenshot 2024-11-07 at 12 34 46" src="https://github.com/user-attachments/assets/3e222e39-8e7a-453b-802d-2d2bd61692a9">

Reply:
We tend to change the themes around every once in a while so if they happen to be the same now, that's no guarantee.
We also might want to use these icons somewhere else later with a different color.
So if it works to use `currentColor` I think we should do that.

Reply:
this will require us to wrap the icon in a span to apply the color. something like:
```
<span class="yes">
    <IconThumbUpCircle size="48" />
</span>
```
are we ok with this?

Reply:
Yes, I think we almost always do this with icons.